src/pubsub/index.js

```diff
@@ -696,6 +730,16 @@ class PubsubBaseProtocol extends EventEmitter {
   }
 }
 
+/**
+ * @typedef {any} Libp2p
+ * @typedef {import('./peer-streams').DuplexIterableStream} DuplexIterableStream
+ * @typedef {import('../connection/connection')} Connection
+ * @typedef {import('./message').RPC} RPC
+ * @typedef {import('./message').SubOpts} RPCSubOpts
+ * @typedef {import('./message').Message} RPCMessage
+ * @typedef {import('./signature-policy').SignaturePolicyType} SignaturePolicyType
+ */
+
 module.exports = PubsubBaseProtocol
 module.exports.message = message
 module.exports.utils = utils
```

Comment:
its better to always use named exports especially in this case where default and named are mixed!